### DISCUSSION TITLE
Show your favorite stations to the world !

### DISCUSSION THREAD

**Discussion by deep5050** [deep5050/radio-active#10] [Show and tell]
Share your favorite stations with us :)
Others may be searching for it

**Comment by deep5050** [comment]
pehla_nasha==https://wmirchi-lh.akamaihd.net/i/WMIRCHI_1@75780/master.m3u8
hello_fm==6829847d-14f5-42aa-82e3-9ac04846e0b4
purani_jeans==https://puranijeanshdliv-lh.akamaihd.net/i/PuraniJeansHDLive_1_1@334555/index_1_a-p.m3u8?sd=10&rebase=on
club==https://clubmirchihdlive-lh.akamaihd.net/i/ClubMirchiHDLive_1_1@336269/master.m3u8
ishq==https://ishq_kolkata-lh.akamaihd.net/i/Ishq_Kolkata_1@854569/master.m3u8
ishq_kolkata==https://ishq_kolkata-lh.akamaihd.net/i/Ishq_Kolkata_1@854569/master.m3u8
NDTV==https://ndtv24x7elemarchana.akamaized.net/hls/live/2003678/ndtv24x7/ndtv24x7master.m3u8
kalighat_radio==https://onlineradiobox.com/json/in/kalighatradio/play?platform=web
hungama_evergreen_bollywood==https://onlineradiobox.com/json/in/evergreenbollywood/play?platform=web
hungama_edm==https://onlineradiobox.com/json/in/bestofedm/play?platform=web
hungama_artistcould==https://onlineradiobox.com/json/in/artistaloud/play?platform=web
hungama_indiepop==https://onlineradiobox.com/json/in/bestofindiepop/play?platform=web
hungama_dilse==https://onlineradiobox.com/json/in/bollywooddilse/play?platform=web
radio_kantipur_nepal==https://radio-broadcast.ekantipur.com/stream
radio_bhumi_bangladesh==https://radiobhumilive.radioca.st/;stream.mp3
radio_dhaka==https://onlineradiobox.com/json/bd/dhaka/play?platform=web
radio_city_91.9==https://prclive1.listenon.in/
hits_of_bollywood==https://stream.zeno.fm/60ef4p33vxquv
bongonet==https://www.bongonet.net/bongonet

**Comment by Yasumoto** [comment]
```
The Trip==960eb52f-0601-11e8-ae97-52543be04c81
```

[The Trip on SomaFM](https://somafm.com/thetrip/index.html)!

**Reply by deep5050** [reply]
This is an amazing station.. thanks,,very much :)

**Comment by thabomakwea** [comment]
[Beat Radio Extra](https://radiotub.com/asia/radio/Radio_Beat_Extra)
[Amapiano FM](https://zeno.fm/radio/amapiano/)
[Radio 105 - 105 Hip Hop & R'N'B](https://radiotub.com/europe/radio/105_Hip_Hop)

**Comment by tangowhisky37** [comment]
Hey mate... this is one helluva CLI radio player!!!

Excellent work mate. Thanks for sharing.
